OpenRA.Game/Network/Order.cs

```diff
@@ -193,9 +197,6 @@ public static Order Deserialize(World world, BinaryReader r)
 						if (world == null)
 							return new Order(order, null, target, targetString, queued, extraActors, extraLocation, extraData, groupedActors);
```

Comment:
Now that you moved the subject check this order will not get returned. If there's no world there's no subject, and as you can see it gives a perfectly viable order with no subject

Reply:
don't notice that, reverted